odysseus/convertFtbQuests.ts

```diff
@@ -383,6 +383,10 @@ function truncateLong(value: Long | undefined) {
     return Number(value)
 }
 
+function iconBackgroundTexture(iconBackground: QuestShape | undefined): ResourceLocation | undefined {
+	return iconBackground === undefined ? undefined : `heracles:textures/gui/quest_backgrounds/${iconBackground}s.png`
```

Comment:
```suggestion
	if (iconBackground === 'square') return "heracles:textures/gui/quest_backgrounds/default.png"
	return iconBackground === undefined ? undefined : `heracles:textures/gui/quest_backgrounds/${iconBackground}s.png`
```

Reply:
I'd suggest using a variable, and setting it to `default` in that case and using it as the templated parameter for the texture name. That'd reduce the amount of times the full texture path is typed.

Reply:
that'd be `defaults.png`

Reply:
Can be set to `default` or `${iconBackground}s` depending on the branch.

Reply:
swapped the suggestion to use undefined instead, considering heracles considers that equivalent.